### ISSUE TITLE
Stabiliteit van eventstore-sync-multichain onderzoeken

### ISSUE THREAD

**Issue by marcvanandel** [kadaster-labs/sensrnet-home#212]
Het lijkt erop dat de koppeling van de eventstore (waar al een issue voor is om die te upgraden) naar de sync en dan naar de multichain pod niet altijd goed draait. De sync wordt met regelmaat herstart omdat een connectie wegvalt. Dit moet beter worden en de oorzaak van het wegvallen moet onderzocht worden

**Comment by marcvanandel** [comment]
Hmm... de multichain node is niet helemaal stabiel. Dit haal ik uit de logging in de test env:

```
Start node with existing chain: SensRNet from 20.71.84.87:8571
Chain SensRNet already exists, adding 20.71.84.87:8571 to list of peers

Other nodes can connect to this node using:
multichaind SensRNet@10.244.1.25:8571

Node didn't shut down normally, performing recovery

Node ready.
```

This is recovery of the multichain process within the pod, right?

**Comment by kad-floriw** [comment]
Yes dat is het recovery process binnen de pod. Dat krijg je als de vorige pod abrupt is afgesloten. Ik weet niet wat de oorzaak is van het abrupt afsluiten.

**Comment by kad-busses** [comment]
De melding is altijd zichtbaar wanneer de pod gestopt is en opnieuw wordt gestart. Zo krijg je 'm bijvoorbeeld ook altijd binnen Docker Compose te zien wanneer de `main-node` opnieuw gerund wordt.

In dit specifieke geval is de oorzaak is dat de Multichain deployment is geupdate naar de laatste versie 😉

**Comment by marcvanandel** [comment]
Zou de Sync pod direct down moeten gaan of kan dat wat vriendelijker?
